src/coreclr/jit/instrsriscv64.h

```diff
@@ -0,0 +1,247 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*****************************************************************************
+ *  LoongArch64 instructions for JIT compiler
```

Comment:
Forgot to change the reference of LoongArch64 to riscv64.

Reply:
I implemented based on LoongArch, but I didn't update some codes and comments.
Thank you. I updated all.